{"name":"Проверка поведения сервиса при ошибке базы данных","status":"passed","stage":"finished","description":"Тест проверяет поведение сервиса при возникновении ошибок доступа к базе данных.","steps":[],"attachments":[],"parameters":[],"start":1713552010105,"stop":1713552010111,"uuid":"d1b83f64-c401-4b89-93b0-227aac3c6351","historyId":"b82ab12a0a74651fa04abafa65689fc7","fullName":"searchengine.services.statistics_service.StatisticsServiceDatabaseErrorTest.testGetStatisticsWithDatabaseError","labels":[{"name":"feature","value":"Обработка ошибок базы данных"},{"name":"epic","value":"Тестирование сервиса статистики"},{"name":"owner","value":"Соминский Антон"},{"name":"story","value":"Обработка ошибок базы данных"},{"name":"host","value":"DESKTOP-8T1LS71"},{"name":"thread","value":"17272@DESKTOP-8T1LS71.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"searchengine.services.statistics_service.StatisticsServiceDatabaseErrorTest"},{"name":"testClass","value":"searchengine.services.statistics_service.StatisticsServiceDatabaseErrorTest"},{"name":"testMethod","value":"testGetStatisticsWithDatabaseError"},{"name":"suite","value":"searchengine.services.statistics_service.StatisticsServiceDatabaseErrorTest"},{"name":"severity","value":"critical"}],"links":[]}